pages/mentoring_faqs.md

```diff
@@ -3,6 +3,9 @@ Here we have collated a list of the questions asked frequently by mentors.
 ### How do I get started with Mentoring?
 Once you've become a mentor via the [become a mentor](https://exercism.io/become-a-mentor) page, you can access your [mentor dashboard](https://exercism.io/mentor/dashboard) at any time. When you joined you should have received an invitation to our Slack Room. You can find more instructions in the [start-here channel on Slack](https://exercism-mentors.slack.com/archives/CBDA9LMDF/p1529968977000336).
 
+### What is the goal of mentoring a solution?
+The purpose of Exercism is to improve programming language _fluency_ rather than proficiency. In this sense, fluency is the ability to "speak" the language, to read and write code in it that communicates with other programmers and the machine. Proficiency is the ability to write code that displays good style and technical rigor.
```

Comment:
```suggestion
The purpose of Exercism is to improve programming language _fluency_ rather than proficiency. In this sense, fluency is the ability to "speak" the language, to read and write code in it that communicates with other programmers and the machine. Our goal is to teach someone to write code in that language like a "native" programmer in that language would. Proficiency is the ability to write real-world programs using a language - e.g. understanding the standard library, the ecosystem, displaying technical rigor or algorithmic competency.
```

Reply:
The proposed change looks good to me, @workingjubilee ?

Reply:
Ah, this looks fine. I would be inclined to merge the two sentences on fluency rather than preserving my own untouched, because it feels like both rehearse a similar "talking point"?

Reply:
If you are fine with @iHiD's suggestion then you just need to click `Commit suggestion`, after that we can see about merging.